pravega-client-bytestream-api-101/src/main/java/io/pravega/TimeLapse.java

```diff
@@ -0,0 +1,123 @@
+package io.pravega;
+
+import io.pravega.client.ByteStreamClientFactory;
+import io.pravega.client.ClientConfig;
+import io.pravega.client.admin.StreamManager;
+import io.pravega.client.byteStream.ByteStreamReader;
+import io.pravega.client.byteStream.ByteStreamWriter;
+import io.pravega.client.stream.StreamConfiguration;
+import lombok.Cleanup;
+
+import javax.imageio.ImageIO;
+import javax.imageio.ImageReader;
+import javax.imageio.stream.ImageInputStream;
+import javax.swing.*;
+import java.awt.image.BufferedImage;
+import java.io.File;
+import java.io.IOException;
+import java.net.URI;
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Map;
+
+public class TimeLapse {
+
+    final static String scopeName = "scope2";
+    final static String streamName = "ndvu";
+    final static String CONTROLLER_IP = "tcp://127.0.0.1:9090";
```

Comment:
Maybe this is beyond the scope of this pr. But it would be good to change this default because 9090 is used for a lot of other things.

Reply:
@tkaitchuck -- Relatedly, should Pravega find service names and port numbers in the User Port range that it could conceivably take to IANA one day for assignment?